Library/Formula/cycript.rb

```diff
@@ -0,0 +1,93 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git", :tag => "v0.9.592", :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
+
+  def install
+    ENV.append "CPPFLAGS", "-Wno-inconsistent-missing-override"
+    ENV.append "CPPFLAGS", "-I#{HOMEBREW_PREFIX}/opt/readline/include"
```

Comment:
These two lines (The `readline` ones) shouldn't be necessary. Should happen automatically.